client/views/app/sideNav/directMessagesFlex.coffee

```diff
@@ -36,6 +37,12 @@ Template.directMessagesFlex.helpers
 	selectedUser: ->
 		return Template.instance().selectedUser.get()
 
+	warning: ->
+		if Template.instance().warning.get()
+			return 'Due to security label access conflicts, you will not be able to proceed with the currently-selected labels.'
```

Comment:
Ah, that is incorrect.. I got too hyphen-happy ;). According to Rule 3 here: (http://www.grammarbook.com/punctuation/hyphens.asp), the hyphen shouldn't be there for adverbs ending in -ly. I think there might be other instances of this - I'll fix 'em.